Networking/Networking/Extensions/DateFormatter+Woo.swift

```diff
@@ -19,6 +19,15 @@ public extension DateFormatter {
             return formatter
         }()
 
+        /// Date And Time Formatter for simple dates — e.g. `2019-02-01` or `2019-11-28`
+        ///
+        public static let simpleDateTimeFormatter: DateFormatter = {
```

Comment:
This formatter is setup with the `en_US_POSIX` locale. If I understand correctly, that's because the extension returns dates formatted that way, so it makes sense.

To me, and this might be my lack of general knowledge about the project and the codebase, the name of the formatter does not seem to carry that context. It is true that the headerdoc provides an example of the expected format, but would it make sense to make the name reflect either the locale or the expected format?

Reply:
Fair point — I will update the name/inline doc.